routers/web/repo/pull.go

```diff
@@ -1012,10 +1012,14 @@ func CompareAndPullRequestPost(ctx *context.Context) {
 	)
 
 	headUser, headRepo, headGitRepo, prInfo, baseBranch, headBranch := ParseCompareInfo(ctx)
+	defer func() {
```

Comment:
This was fin but I'm not going to argue

Reply:
Yes it is in the refactor commit ...

Make it more redable